ocaml/xapi/sm.ml

```diff
@@ -210,15 +219,17 @@ let vdi_activate dconf driver sr vdi writable =
   in
   Sm_exec.parse_unit (Sm_exec.exec_xmlrpc (driver_filename driver) call)
 
-let vdi_deactivate dconf driver sr vdi =
+let vdi_deactivate ?dbg dconf driver sr vdi =
```

Comment:
Is there a reason for the optional argument here? Is it because of the call in static_vdis.ml? I think we should change _that_ one to add a `~dbg` to the call with

    let dbg = Context.string_of_task_and_tracing __context in

Reply:
I see that Marcus had suggested this in a previous comment. However, I think it is easy to construct a `dbg` as above in static_vdis.ml. This ends up making tracing more complete.